packaging/build_deb

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -xe
+# Execute this shell-script at root directory of libsxg project.
+
+SRCDIR=$(pwd)
+WORKDIR=$(mktemp -d)
```

Comment:
Optional: Consider a non-default template, i.e. `mktemp --tmpdir -d something.X…X`.

https://manpages.debian.org/buster/coreutils/mktemp.1.en.html

Reply:
Interesting but I have no reason to justify using it.